JS/cart.js

```diff
@@ -0,0 +1,49 @@
+// Render Cart
+function removeInCart(arr, id) {
+    const myArr = [...arr];
+    let idx = myArr.findIndex((e) => {
+        if (e.id == id)
+            return true;
+    });
+    if (idx == -1) return;
+    myArr.splice(idx, 1);
+    return myArr;
+}
+
+function getCArtItems() {
+    const ArrayofCartObjects = JSON.parse(localStorage.getItem('ArrayofCartObjects') || '[]');
+    console.log(ArrayofCartObjects)
+    const table = document.querySelector('#cart tbody')
+    table.innerHTML = '';
```

Comment:
beware, you can use native methods that aren't safe like `innerHTML` please read more here about [safely inserting external content into a page](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Safely_inserting_external_content_into_a_page#working_with_arbitrary_strings)